extensions-contrib/time-min-max/src/main/java/io/druid/query/aggregation/TimestampAggregatorFactory.java

```diff
@@ -86,6 +87,12 @@ public Object combine(Object lhs, Object rhs)
   }
 
   @Override
+  public AggregateCombiner makeAggregateCombiner()
+  {
+    throw new UOE("[%s] is not supported during ingestion for rollup", getClass().getSimpleName());
```

Comment:
the timestamp min/max aggs can be used to aggregate columns other than `_time`:

https://github.com/druid-io/druid/blob/master/docs/content/development/extensions-contrib/time-min-max.md

>To use this feature, a "timeMin" or "timeMax" aggregator must be included at indexing time. They can apply to any columns that can be converted to timestamp, which include Long, DateTime, Timestamp, and String types.